etc/picongpu/juwels-jsc/booster_picongpu.profile.example

```diff
@@ -0,0 +1,126 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+#   - using NF - number of fields, use the last group available
+#export proj=$(groups | awk '{print $(NF)}')
+export proj=ea_picongpu
+# Text Editor for Tools ###################################### (edit this line)
+#   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
+#export EDITOR="vim"
+
+# Set up environment, including $SCRATCH and $PROJECT
+jutil env activate -p $proj
+
+# General modules #############################################################
+#
+# Requires module stages to be loaded
+OTHERSTAGES=/p/software/juwelsbooster/otherstages
+module purge
+
+module use $OTHERSTAGES
+module load Stages/2020
+
+module load GCC/9.3.0
+module load CMake/3.18.0
+module load ParaStationMPI/5.4.7-1
+module load mpi-settings/default
+#module load mpi-settings/CUDA 
+module load Python/3.8.5
+
+module load Boost/1.74.0
+module load HDF5/1.10.6
+echo "Loaded modules"
+# Other Software ##############################################################
+#
+
+# Define the path to manually installed libraries
+PARTITION_LIB=$PROJECT_boosterea/lib
+
+LIBSPLASH_VER=1.7.0
+PNGWRITER_VER=0.7.0
+BLOSC_VER=1.19.0
+ADIOS_VER=1.13.1
+
+LIBSPLASH_ROOT=$PARTITION_LIB/LIBSPLASH/$LIBSPLASH_VER
+PNGWRITER_ROOT=$PARTITION_LIB/PNGWRITER/$PNGWRITER_VER
+export CMAKE_PREFIX_PATH=$LIBSPLASH_ROOT:$PNGWRITER_ROOT:$CMAKE_PREFIX_PATH
+
+BLOSC_ROOT=$PARTITION_LIB/BLOSC/$BLOSC_VER
+export CMAKE_PREFIX_PATH=$BLOSC_ROOT:$CMAKE_PREFIX_PATH
+export LD_LIBRARY_PATH=$BLOSC_ROOT/lib:$LD_LIBRARY_PATH
+
+ADIOS_ROOT=$PARTITION_LIB/ADIOS/$ADIOS_VER
+export PATH=$ADIOS_ROOT/bin:$PATH
+export CMAKE_PREFIX_PATH=$ADIOS_ROOT:$CMAKE_PREFIX_PATH
+
+
+# Environment #################################################################
+#
+
+export PICSRC=$HOME/shared/src/picongpu-dev-04092020
```

Comment:
This is quite specific (probably your directory name) - a more generic name would be better - should be fixed with the next update (not urgent)

Reply:
For a more general usage sure. At the same time the project variable will have to be fixed as well, since this profile is currently intended more to have a shared profile for the EA.